pajbot/modules/eightball.py

```diff
@@ -109,7 +115,7 @@ def eightball_command(self, **options):
         message = options['message']
 
         if message and len(message) > 0:
-            phrase = random.choice(self.phrases)
+            phrase = random.choice(self.phrases) + " " + random.choice(self.emotes)
```

Comment:
Using string concatenation like that is generally frowned upon.
You should instead keep phrase as it was before:
`phrase = random.choice(self.phrases)`
and then do
`emote = random.choice(self.emotes)`
and change the bot.me message to include {phrase} and then {emote} instead